src/watertap_contrib/seto/solar_models/surrogate/trough/data/training_trough_surrogate.py

```diff
@@ -154,23 +157,32 @@ def plot_training_validation(
         )
 
         if save_figs is not None:
-            plt.savefig("/plots/parity_residual_plots.png")
+            plt.savefig(Path(__file__).parent / "plots" / "parity_residual_plots.png")
             plt.close()
 
 
 #########################################################################################################
 if __name__ == "__main__":
-    dataset_filename = join(dirname(__file__), "trough_data.pkl")
-    surrogate_filename = join(dirname(__file__), "trough_surrogate_testing.json")
+    heat_load_range = (10, 100)         # must be (10, 100) or (100, 500)
```

Comment:
Maybe a comment here guiding the user to update the range here and the scaling dict in trough_surrogate?

Reply:
I'd view this as more a demonstration rather than a final implementation. Ultimately we will have to make new surrogates anyway for a different location.